extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlCordon.java

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl;
+
+import io.kubernetes.client.custom.V1Patch;
+import io.kubernetes.client.extended.kubectl.exception.KubectlException;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1Node;
+import io.kubernetes.client.util.PatchUtils;
+
+public class KubectlCordon extends Kubectl.ResourceAndContainerBuilder<V1Node, KubectlCordon>
+        implements Kubectl.Executable<Boolean> {
```

Comment:
```suggestion
        implements Kubectl.Executable<V1Node> {
```

normally we consider the method to be success if it doesn't throw a `KubectlException` is it? how about returning the patched node here?